deploy/WMAgent.production

```diff
@@ -27,4 +27,12 @@ RUCIO_HOST=http://cms-rucio.cern.ch
 RUCIO_AUTH=https://cms-rucio-auth.cern.ch
 TEAMNAME=
 AGENT_NUMBER=
-MSPILEUP_URL=https://cmsweb.cern.ch/ms-pileup/data/pileup
```

Comment:
hi @anpicci  was  this line removal intentional?

Reply:
No, I can re-add it if necesseary @todor-ivanov

Reply:
please do :) Thanks!

Reply:
no, I will add it back, thank you